includes/functions.php

```diff
@@ -2756,6 +2816,7 @@ function dokan_cache_clear_seller_product_data( $product_id, $post_data = [] ) {
 
     dokan_clear_product_caches( $product_id );
     dokan_cache_clear_group( 'dokan_seller_product_data_' . $seller_id );
+    dokan_cache_clear_group( 'dokan_seller_product_stock_data_' . $seller_id );
```

Comment:
I'm sorry to say, but this will not work until we store cache key in option table :(